src/vextab_parsing.py

```diff
@@ -25,7 +25,17 @@
     2: (constants.REVERSE_SYMBOLS["B"], 3),
     1: (constants.REVERSE_SYMBOLS["E"], 4)
 }
-# print(STRINGS)
+
```

Comment:
This file is here bc I forked from https://github.com/JakubBilski/tonations-recognition/pull/84, not from master. Please ignore it